gulpfile.js

```diff
@@ -10,37 +10,13 @@
  */
 
 var gulp = require('gulp');
-var execSync = require('child_process').execSync;
-
-var packageJson = require('./package.json');
 
 var typings = require('./scripts/gulpfiles/typings');
 var buildTasks = require('./scripts/gulpfiles/build_tasks');
 var packageTasks = require('./scripts/gulpfiles/package_tasks');
 var gitTasks = require('./scripts/gulpfiles/git_tasks');
 var licenseTasks = require('./scripts/gulpfiles/license_tasks');
 
-// See https://docs.npmjs.com/cli/version.
-const preversion = gulp.series(
```

Comment:
You removed these two (and their entries in package.json) because now we can use the builtin versions, right?

Reply:
I did it because these are creating a bump_version branch that I don't think we will need anymore since we are combining bumping the version with our recompile script.